src/neon/how-tos/advance-analysis/undistort.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Undistorting Video and Gaze Data
+description: "Undistorting Neon's scene camera video and gaze data, using the intrinsic and extrinsic camera parameters."
+permalink: /neon/how-tos/advance-analysis/undistort/
+---
+# Correcting for the fish-eye lens distortion on Neon's scene camera
+One particular challenge that often arises in eye tracking setups is the presence of [fish-eye lens distortions](https://en.wikipedia.org/wiki/Fisheye_lens), also known as barrel distortions. Fish-eye lenses, with their wide-angle characteristics, allow for a broad field of view, enabling the capture of more information within a single frame. However, this advantage comes at the cost of introducing non-linear distortions. 
+
+While we account for them in Pupil Cloud, and we even give you the possibility to download the undistorted video using the [Gaze Overlay enrichment](/enrichments/gaze-overlay), this is not the case when using the [Realtime API](/neon/real-time-api) or raw data, where you may want to correct it by yourself.
+
+## Why is it important to correct for fisheye lens distortions?
+Correcting these distortions brings many benefits, by aligning the captured scene with the underlying geometry of the environment, the corrected images provide a more faithful representation of the observer's visual perspective. This alignment is particularly crucial in scenarios where spatial relationships, distances, or areas of interest are essential factors under investigation.
+
+Moreover, the correct calibration of fisheye lens distortions ensures comparability and consistency across different eye tracking studies. Researchers often rely on datasets collected from various experiments, and without a standardized correction procedure, variations in lens distortions could introduce unnecessary variability and hinder the ability to generalize findings across studies.
+
+Beyond the scientific realm, the application of fisheye lens correction extends to practical domains as well. If you plan to incorporate Neon to virtual reality or augmented reality, you will need to affix the Neon module to the headset and to compute the geometrical relationship from the scene camera to the virtual reality camera. By accounting for fisheye lens distortions, you can better correlate the gaze position with the underlying geometry of the virtual environment, thus improving the accuracy of your gaze-based interactions.
+
+## How to correct for fisheye lens distortions?
+In this tutorial, we will cover how to read the provided intrinsic and extrinsic camera parameters and correct for fisheye lens distortions in Neon's scene camera. We will also show you how to undistort the gaze data, so that you can map the gaze position onto the undistorted scene camera video.
```

Comment:
The extrinsics usually describe how a camera is positioned in the world, which we don't really have, do we? As far as I can tell we are only using intrinsics.

Reply:
Then we would probably want to rename some fields like:

"scene_extrinsics_affine_matrix",